.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: false
+language: python
+python: 3.6
+cache: pip
+fast_finish: true
+branches:
+  only:
+  - master
```

Comment:
hmm, in development phase i usually run sceptre manually to test that it works but i see that's gonna be a problem for you.  let me give it some thought.

Reply:
Do you mean you're running `aws cloudformation create-stack`? Because running sceptre myself, even in sandbox, didn't work, as I didn't have the necessary permissions even with the admin role. So that would test creating the stack but not whether sceptre is going to fail.

Reply:
I should add, right now sceptre is pretty much a black box to me. Perhaps you have more confidence in trusting it will do what it's supposed to.

Reply:
yes, that is correct.  ahh yes, it's not working for you because you don't have access to the needed secrets.   I can and will fix that.

Not sure what you are eluding to about Sceptre but I can tell you that we are using it to deploy all of our infra across 10 different AWS accounts right now so yeah, it works pretty well.

Reply:
I only mean that I personally don't know, and my build was failing and I didn't know why. So you are recommending that I run create-stack? I'm going to go ahead and approve this since you have a workaround in mind for me.